Scripts/parameters/demand/hb_edu_student.json

```
@@ -97,11 +97,20 @@
             },
             "log": {
                 "attraction_size": 1,
-                "cost": -0.28318
+                "cost": -0.28318,
+                "transform": -0.1
```

Because "cost" has been moved to "transform", it should not be here as well. We also need to apply the correct parameter.

```suggestion
                "transform": -0.28318
```